dotcom-rendering/playwright/lib/islands.ts

```diff
@@ -4,36 +4,34 @@ type IslandStatus = 'rendered' | 'hydrated';
 type WaitForIslandOptions = {
 	status?: IslandStatus;
 	nth?: number;
+	waitFor?: 'attached' | 'detached' | 'visible' | 'hidden';
```

Comment:
What about `state`?

Reply:
Might be confusing with the island 'status'.

Could do `waitForState`

Reply:
Status is going away very soon: #allislands